### ISSUE TITLE
Get all connected users by room id

### ISSUE THREAD

**Issue by amanpoplify** [andyet/SimpleWebRTC#713]
How can i get lists of all connected users by their room id?

**Comment by hokuco** [comment]
yes, you can, like this:(this should be a tad buggy, I just made it here for you)
`
var list[];
webrtc.on('createdPeer', function (peer) {
    var pyd = peer.id
list.push(pyd);
}`
